uNmAnNeR/imaskjs
# feat: solid-imask implementation proposal

**Work Done**
-
- Proposal for the solid-imask package

**To Test**
-
- npm run build in the packages/solid-imask folder
- copy /dist+package.json to node-modules of a solid project
- make sure the project has i-mask pre-installed
- test the api and type inference
